Backend/ManagerLayer/UserManagement/UserManagementManager.cs

```diff
@@ -141,5 +144,13 @@ public int UpdateUser(User user)
                 }
             }
         }
+
+        public bool ExistingUser(string email)
```

Comment:
Check the user repository - does this/a similar method already exist? I'm not sure, but probably best to check on that.